pkgs/by-name/dt/dte/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dte";
+  version = "1.11.1";
+
+  src = fetchFromGitLab {
+    owner = "craigbarnes";
+    repo = "dte";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-PG9yfRzJKXVKAg8ubIvX2Rj4nLO3dki7shP+zAa7nxo=";
+  };
+
+  installFlags = [ "DESTDIR=${placeholder "out"}" ];
+
+  postInstall = ''
+    mv $out/usr/local/* $out
+    rm -rf $out/usr
+  '';
+
+  doInstallCheck = true;
+
+  installCheckPhase = with finalAttrs; ''
+    runHook preInstallCheck
+
+    $out/bin/${meta.mainProgram} -V | grep ${version}
```

How will this work when `version` is overridden?
```suggestion
    $out/bin/dte --version
```

---

```sh
$out/bin/dte --version
```

How would this assert the version?

---

`versionCheckHook` has to be in `nativeInstallCheckInputs` then?